generators/server/templates/src/main/java/package/service/mapper/_UserMapper.java

```diff
@@ -54,7 +53,7 @@ default User userFromId(<% if (databaseType == 'mongodb' || databaseType == 'cas
     }
 
     default Set<Authority> authoritiesFromStrings(Set<String> strings) {
-        return strings.stream().map(string -> {
+        return strings == null ? null : strings.stream().map(string -> {
```

Comment:
Still, that feels wrong to me to have a `null` hanging around here -> I need to check this. Could you split your PR in 2? We could merge the rest of this PR, and then have a specific PR for this "null" issue, and discuss this more properly. Besides, I'm not sure Sonar will like it, and we work hard to have the best Sonar rating at https://sonarqube.com/dashboard/?id=1054756